README.md

```diff
@@ -121,7 +120,6 @@ installed:
  - libgetopt-java (GNU getopt)
  - Link Parser
  - WordNet 3.0
- - JWNL Java wordnet library
  - OpenNLP tools (optional, but recommended)
  - W3C OWL (optional)
```

Comment:
`libgetopt-java`, `OpenNLP tools` and `W3C OWL` and `***GNU getopt***` section below could be removed as well

Reply:
Updated.

Reply:
Well, somewhere, some file should explain what the dependencies are ... it could say "these are needed, you don't have to install them manually, they are automatically installed by maven, but they are needed and used"

Reply:
Returned tools installed by Maven back to README file.

Reply:
I don't think we need dependencies description in README.md:
- one can find list of dependencies in `pom.xml`, see https://github.com/opencog/relex/blob/2f71dc0f62a83d59ec3677d2c64a7b77813b93af/pom.xml#L246
- one can find theirs descriptions in maven repository.

Especially common libraries like slf4j, apache-commons, logback which are well known for each Java developer.
Getopt and Trove are not used directly.